core/sail/fts/elasticsearch/pom.xml

```diff
@@ -47,11 +49,35 @@
 			<version>${project.version}</version>
 			<scope>test</scope>
 		</dependency>
-		<dependency>
-			<groupId>org.slf4j</groupId>
-			<artifactId>slf4j-simple</artifactId>
-			<version>${slf4j.version}</version>
-			<scope>test</scope>
-		</dependency>
+                <dependency>
```

Comment:
I'd prefer instead switching to logback, and including a logback-text.xml config file in src/test/resources, like it's done in other parts of the code base. Makes things more consistent, and also avoids adding new dependencies (and having to bump the jackson version).